templates/patterns/card/pattern-card.html.twig

```diff
@@ -20,7 +20,7 @@
     {{ content }}
   {% endset %}
   {% if _rendered_content|trim is not empty %}
-    <div>{{ _rendered_content }}</div>
+    <div class="mt-2-5">{{ _rendered_content }}</div>
```

Comment:
so what if I send only content now, without meta? we get the same situation as reported but in a different combination of fields set. We need to resolve this without introducing new issues.

Reply:
now it should not be a problem because the margin is top... also the margin between elements are eaten by themselves and only the bigger one is kept ex mb-3 vs mb-2-5 - the margin will be 16px (mb-3)